packages/docs/celo-holder-guide/ledger.md

```diff
@@ -169,3 +153,18 @@ If you have issues connecting to the Ledger, try the following:
 - Ensure that you are running the latest version of the Celo CLI.
 
 There have been reports of a possible [issue](https://github.com/celo-org/celo-ledger-spender-app/issues/13) that appears to affect developer store apps on the Ledger Nano X including the Celo Ledger App. This is believed to be fixed in version 1.0.3. In earlier versions, a user clicking through the `Pending Ledger review` notice too rapidly can cause the device to freeze. If this occurs, wait until the device's battery is depleted, then charge and power up again. Then use Ledger Live Manager to update the installed version of the Celo Ledger App.
+
+## Using Your Ledger with Celo Wallet app
+
+An alternative to using `celocli` is [celowallet.app](https://celowallet.app)
+
+1. To get started using it with your Ledger, go to the wallet [here](https://celowallet.app)
+2. Click on `Use Existing Account` option.
+3. Click on the Ledger icon.
+4. Connect your Ledger Nano to your computer.
+5. Unlock your Ledger Nano with your PIN.
+6. Go to the Celo app on your Ledger Nano and open it.
+7. On `celowallet.app` website, click on `Import Account`. Make sure you are using the right Address Index. Index 0 means the first address on the list.
+8. There will be a popup in on the website to approve the Ledger device, click on Nano in the popup and click `Connect`.
```

Comment:
Nit: the permission popup is from the browser, not the website. This is slightly relevant since it's positioning and style vary between browsers